assets/blocks/course-list-block/course-list-block-editor.scss

```diff
@@ -5,3 +5,11 @@
 		}
 	}
 }
+.block-editor-block-pattern-setup.view-mode-grid {
+	.block-editor-block-pattern-setup__grid {
+		.block-editor-block-pattern-setup__container.course-list-pattern-picker {
```

Comment:
```suggestion
	.block-editor-block-pattern-setup__container { 
	 &.course-list-pattern-picker {
```

Reply:
Hi @gabrielcaires , thanks for the suggestion. Have you tried this locally and has it worked for you? 🤔 Though you are 100% right theoretically and it looks better, there's another selector that gets prioritized for setting the `display` property. That's why I had to kind of overspecify to increase the priority of this style, as a way around of using `!important`

Reply:
Yes, I am seeing the same 🤔.
I am going to approve it.